src/python/pants/option/global_options.py

```diff
@@ -915,13 +915,23 @@ def register_options(cls, register):
             "tags ('-' prefix).  Useful with ::, to find subsets of targets "
             "(e.g., integration tests.)",
         )
+        register(
+            "--dynamic-ui",
+            type=bool,
+            default=sys.stdout.isatty(),
```

Comment:
This means that we are going to default to V2 UI being on for everyone. I think we're really close to doing that, but we should save it for a followup after having had fixed the couple remaining known bugs.

Reply:
Which are you referring to? I'm not aware of any bugs caused by the UI. If you're thinking of signal handling (ie, Ctrl+C), that is handled independently from the UI: I don't think that the UI actually changes how we handle signals at all.

Reply:
@gshuflin : The UI renders on stderr, I think? If so, this should be checking `stderr` rather than `stdout`.

Reply:
Some bugs Greg has been working on. 

1. `cloc` doesn't properly output the first run, but does the second.
2. `test --debug` keeps the UI around permanently.

Reply:
Got it. Yea, those probably qualify as blockers. Would be good to file/track them.